pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/LookupProxyHandler.java

```diff
@@ -54,6 +57,8 @@ public LookupProxyHandler(ProxyService proxy, ProxyConnection proxyConnection) {
         this.proxyConnection = proxyConnection;
         this.clientAddress = proxyConnection.clientAddress();
         this.connectWithTLS = proxy.getConfiguration().isTlsEnabledWithBroker();
+        this.brokerServiceURL = this.connectWithTLS ? proxy.getConfiguration().getBrokerServiceURLTLS()
```

Comment:
should we also add another check
`this.connectWithTLS && isNotBlank(proxy.getConfiguration().getBrokerServiceURLTLS()) ? proxy.getConfiguration().getBrokerServiceURLTLS() : proxy.getConfiguration().getBrokerServiceURL()` ?